[{"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/MT_NRCS_B2_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/MT_NRCS_B2_2018/ept-data/0-0-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_SouthernNE_B4B_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NE_SouthernNE_B4B_2018/ept-data/1-0-1-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_NorthWest_Navajo_B_TL_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_NorthWest_Navajo_B_TL_2018/ept-data/1-0-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthCentral_B9_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/NM_SouthCentral_B9_2018/ept-data/2-0-1-2.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/OH_Statewide_Phase1_3_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/OH_Statewide_Phase1_3_2019/ept-data/1-1-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/PA_Northcentral_B1_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/PA_Northcentral_B1_2019/ept-data/2-0-1-2.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SavannahPeeDee_7_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SC_SavannahPeeDee_7_2019/ept-data/2-1-1-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_MO-River-Dewberry-B7_2016/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_MO-River-Dewberry-B7_2016/ept-data/1-1-1-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_Spink_County_2012/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/SD_Spink_County_2012/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_HaysCo_2008/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_HaysCo_2008/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/TX_WestTexas_B3_2018/ept.json"], "error": ""}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AL_ADECA_B2_2016_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AL_ADECA_B2_2016_LAS_2019/ept-data/1-1-0-0.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AR_North_Corridor_B3_2017_LAS_2018/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_AR_North_Corridor_B3_2017_LAS_2018/ept.json\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_Central_Valley_2017_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_Central_Valley_2017_LAS_2019/ept-data/2-0-2-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_NoCAL_Wildfires_PlumasNF_B2_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_NoCAL_Wildfires_PlumasNF_B2_2018/ept-data/1-0-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_SoCAL_Wildfires_TL_2018_LAS_2019/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CA_SoCAL_Wildfires_TL_2018_LAS_2019/ept-data/1-1-0-1.laz\n\n"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CT_Statewide_B8_2016/ept.json"], "error": "PDAL: readers.ept: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/USGS_LPC_CT_Statewide_B8_2016/ept.json\n\n"}, {"error": "Could not create geometry because of errors while reading input."}, {"error": "Could not create geometry because of errors while reading input."}, {"tile": "USGS_LPC_TX_East_UTM15_2017_LAS_2018", "error": "('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"}, {"tile": "USGS_LPC_UT_Central_QL1_B1_2018_LAS_2019", "error": "('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"}, {"tile": "USGS_LPC_WA_Western_North_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"tile": "USGS_LPC_WA_Western_South_2016_LAS_2018", "error": "Expecting value: line 1 column 1 (char 0)"}, {"tile": "UT_StatewideCenSouth_1_2020", "error": "('Connection aborted.', OSError(107, 'Transport endpoint is not connected'))"}, {"args": ["pdal", "info", "--all", "--driver", "readers.ept", "--readers.ept.resolution=800.0", "--readers.ept.threads=6", "--filters.hexbin.edge_size=1000", "--filters.hexbin.threshold=1", "https://s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept.json"], "error": "PDAL: readers.ept: Error reading tile: Could not read from s3-us-west-2.amazonaws.com/usgs-lidar-public/WI_Oshkosh_3Rivers_FondDuLac_TL_2018/ept-data/0-0-0-0.laz\n\n"}]